routers/user/home.go

```diff
@@ -83,11 +83,11 @@ func retrieveFeeds(ctx *middleware.Context, ctxUserID, userID, offset int64, isP
 }
 
 func Dashboard(ctx *middleware.Context) {
-	ctx.Data["Title"] = ctx.Tr("dashboard")
+  ctxUser := getDashboardContextUser(ctx)
+	ctx.Data["Title"] = ctxUser.DisplayName() + " " + ctx.Tr("dashboard")
```

Comment:
Sure!
The only thing reason why I left it in was largely to differentiate it from the user/org's profile page but I'm fine either way